bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAsyncAdmin.java

```diff
@@ -86,7 +84,7 @@ public abstract class BigtableAsyncAdmin implements AsyncAdmin {
   private final BigtableHBaseSettings settings;
   private final CommonConnection asyncConnection;
   private final String bigtableInstanceName;
-  private BigtableClusterName bigtableSnapshotClusterName;
+  private String bigtableSnapshotClusterId;
```

Comment:
nit. I dont think bigtable prefix is necessary here...I think its inferred from the context